test/utilities.test.js

```diff
@@ -0,0 +1,21 @@
+
+// IMPORT MODULES under test here:
+// import { example } from '../example.js';
+
+import { randomWare } from '../utilities.js';
```

Comment:
I'm not sure if you want to test your get randomNumber function, you will probably want to test your compareNumbers function.